sqlalchemy_ibmi/base.py

```diff
@@ -784,7 +788,14 @@ def create_connect_args(self, url):
             raise ValueError("Option entered not valid for "
                              "IBM i Access ODBC Driver")
 
-        name_setting = '1' if opts.get('use_system_naming', 'True') == 'True' else '0'
+        try:
+            name_setting = '0' if not util.strtobool(
+                opts['use_system_naming']) else '1'
+        except ValueError:
+            name_setting = '0'
+        except KeyError:
+            name_setting = '1'
```

Comment:
Why are these different? Either case we got an error: either they didn't supply a value or the value was invalid and in both cases we should set it to a default, which should be off (ie. name_setting='1')

Reply:
name_setting might also make more sense to follow the name of the attribute and call it naming.

Reply:
I was thinking along the lines of, if they are including the keyword that it must mean they want to change the default behavior of `True`

Reply:
If we are doing same behavior for both, can we merge the excepts? into just `except Exception as e:`

Reply:
It's generally bad practice to use a general exception like `Exception`, but it seems you can combine them in a tuple: https://docs.python.org/3/tutorial/errors.html#handling-exceptions

```python
try:
     ...
except (KeyError, ValueError):
    name_setting = '0'
```